styles/globals.css

```diff
@@ -9,7 +9,7 @@
 }
 
 * {
-  font-weight: 500;
+  @apply font-normal;
```

Comment:
@isabensusan recommended `font-weight: 400`, so using `font-normal` instead of `font-medium`.